include/picongpu/plugins/openPMD/Json.cpp

```diff
@@ -298,7 +298,7 @@ The key 'select' must point to either a single string or an array of strings.)EN
             auto& hdf5Dataset = config["hdf5"]["dataset"];
             if(!hdf5Dataset.contains("chunks"))
             {
-                hdf5Dataset["chunks"] = "none";
+                // hdf5Dataset["chunks"] = "none";
```

Comment:
What does this line do? / Why was this commented?

Reply:
This should be not part of the PR. with the HDF5 version on our dev system this change is required else HDF5 is deadlocking. I will remove it .

Reply:
fixed